content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/sboms-and-attestations.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "SBOMs and Attestations in Chainguard Enforce"
+type: "article"
+description: "An overview of how Chainguard Enforce handles attestations and SBOMs"
+date: 2023-03-13T15:56:52-07:00
+lastmod: 2023-03-13T15:56:52-07:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 600
+toc: true
+---
+
+In order to fully leverage the capabilities offered by Chainguard Enforce, it's important to establish a process for creating quality *Software Bills of Materials* (SBOMs). An [SBOM](/open-source/sbom/) is a formal record that contains the details and supply chain relationships (such as dependencies) of the components used in building software.
+
```

Comment:
suggest putting a short sentence/para in here that says something like "CG Enforce can capture SBOM data and use it to help you find images and packages in your ecosystem, as well as apply security policies to your runtime environments. To get the best results from these features, SBOMs should be attached to the images in your registry as signed attestations." 

or something like that. just an overview of what this document is solving